docs/examples/promise/10_upgrade_chain/index.js

```diff
@@ -40,7 +37,7 @@ async function main () {
     .sudo(proposal)
     .sign(adminPair, adminNonce)
     .send(({ events = [], status, type }) => {
-      console.log('Transaction status:', type);
+      console.log('Proposal status:', type);
```

Comment:
May as well use the signAndSend shortcut here - this example is actually way too long and looks too complex.

Reply:
Great, the extract await adminNonce about can go as well - at least it helps making the sample more manageable.

Reply:
Ja, that's almost the same in 3 examples, I'm going to change it in all of them